lib/diagnostic.js

```diff
@@ -4,22 +4,40 @@
 // Describe in your own words what a `callback`
 // is in relation to array methods
 
+A callback function affects what is going in another function or array method
+by telling you what to do with every method that you're iterating over.
```

Comment:
Ya close. A callback is just a function that is called by another function. It can be called on each array element to do something to them